fixtures/node-app-pages/package.json

```diff
@@ -7,22 +7,8 @@
 	"scripts": {
 		"check:type": "tsc && tsc -p tests/tsconfig.json",
 		"dev": "npx wrangler pages dev public --port 12345 --node-compat",
-		"test": "npx jest --forceExit",
-		"test:ci": "npx jest --forceExit"
-	},
-	"jest": {
-		"testRegex": ".*.(test|spec)\\.[jt]sx?$",
-		"transform": {
-			"^.+\\.c?(t|j)sx?$": [
-				"esbuild-jest",
-				{
-					"sourcemap": true
-				}
-			]
-		},
-		"transformIgnorePatterns": [
-			"node_modules/(?!find-up|locate-path|p-locate|p-limit|yocto-queue|path-exists|execa|strip-final-newline|npm-run-path|path-key|onetime|mimic-fn|human-signals|is-stream)"
-		]
+		"test": "npx vitest",
+		"test:ci": "npx vitest"
```

Comment:
Why a different script for CI?

Reply:
Just seems to be the pattern we've adopted everywhere. Didn't want to change _too_ much all in this one PR so left as-was.